MAUI/DataForm/layout.md

```diff
@@ -0,0 +1,427 @@
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing editor order in grid rows
+
+The order of the editors in grid row can be changed within a row by using attributes or by handling `GenerateDataFormItem` event.
+
+#### Using attributes
+
+The order of the editors in grid rows can be changed by using the `ItemsOrderInRow` property in [DataFormDisplayOptionsAttribute] attribute. This is applicable only for Grid layout.
+
+{% endhighlight %}
+{% highlight c# %}
+
+this.dataForm.ColumnCount = 2;
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight c# %}
+
+    [DataFormDisplayOptions(RowOrder = 0, ItemsOrderInRow = 1)]
+    public string LastName { get; set; }
+
+    [DataFormDisplayOptions(RowOrder = 0, ItemsOrderInRow = 0)]
+    public string FirstName { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+The order of the editors in grid rows can be changed by using the [ItemsOrderInRow] property in the `DataFormItem`.
+
+{% tabs %}
+{% highlight c# %}
+
+this.dataForm.ColumnCount = 2;
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.RowOrder = 0;
+                e.DataFormItem.ItemsOrderInRow = 1;
+            }
+            if (e.DataFormItem.FieldName == "LastName")
+            {
+                e.DataFormItem.RowOrder = 0;
+                e.DataFormItem.ItemsOrderInRow = 0;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Change label and editor width
+
+The label and editor width can be set proportionally by using [LabelWidth] and [EditorWidth] properties of [DefaultLayoutSettings] class.
+The [Value] and [UnitType] properties of [DataFormItemLength] each return the value and unit type of the corresponding DataFormItemLength, respectively. 
+
+Label and Editor can be specified with proportionate values such as 0.4* and 0.6* when the [DataFormItemLength.UnitType] is 'Star'.
+Additionally, you can specify a defined width for the editor and label, such as "300, 400," and the [DataFormItemLength.UnitType] is "Absolute."
+
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage 
+...
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm">
+        <dataForm:SfDataForm.DefaultLayoutSettings>
+            <dataForm:DataFormDefaultLayoutSettings
+                LabelWidth="0.4*"
+                EditorWidth="0.6*"/>
+        </dataForm:SfDataForm.DefaultLayoutSettings>
+    </dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+
+    this.dataForm.DefaultLayoutSettings.LabelWidth = 300;
+    this.dataForm.DefaultLayoutSettings.EditorWidth = 400;
+
+{% endhighlight %}
```

Comment:
add another c# tab and include star type decration in c#

```
this.dataForm.DefaultLayoutSettings.LabelWidth = new DataFormItemLength(0.3, DataFormItemLengthUnitType.Star);
        this.dataForm.DefaultLayoutSettings.LabelWidth = new DataFormItemLength(0.7, DataFormItemLengthUnitType.Star);
```

Reply:
content added